Tests/LibWeb/Text/input/UIEvents/MouseEvent-bubbling.html

```diff
@@ -0,0 +1,18 @@
+<script src="../include.js"></script>
+<style>
+    * { border: 1px solid black; }
+    .inner { width: 100px; height: 100px; }
+</style>
+<div class="outer"><div class="inner">
+<script>
+    asyncTest(done => {
+        document.querySelector('.outer').addEventListener('click', function() {
+            println('Outer div clicked');
+        });
+        document.querySelector('.inner').addEventListener('click', function() {
+            println('Inner div clicked');
+        });
+        internals.click(50, 50);
+        setTimeout(done, 50);
```

Comment:
invoking done from setTimeout makes test finish later than it possibly could, slowing down test run. instead, could we invoked done when both callbacks are processed?

Reply:
That would result in the test never finishing if it regresses

Reply:
it would not, because headless-browser timeouts each test, so it would be terminated anyway in case it hangs.